scripts/qa_crowbarsetup.sh

```diff
@@ -49,6 +49,14 @@ export cinder_netapp_login
 export cinder_netapp_password
 export localreposdir_target
 export want_ipmi=${want_ipmi:-false}
+export want_mtu_size=${want_mtu_size:-1500}
```

Comment:
this seems redundant, as the default already is 1500
so this line can be dropped